.github/workflows/build.yml

```diff
@@ -16,13 +16,17 @@ jobs:
             fail-fast: false
             matrix:
                 php: [7.4, 8.0]
+                sylius: [~1.9.0, ~1.10.0, ~1.11.0]
                 node: [10.x]
                 mysql: [5.7, 8.0]
 
                 exclude:
                     - # Segmentation fault while clearing cache
                         php: 8.0
                         mysql: 5.7
+                    -
+                        sylius: ~1.11.0
+                        php: 7.4
 
         env:
             APP_ENV: test
```

Comment:
There's no restricting Sylius version. We need to add this. After restricting probably exluding Sylius 1.9 and PHP 8 is going to be needed. Ref workflow https://github.com/BitBagCommerce/SyliusWishlistPlugin/blob/master/.github/workflows/build.yml

Reply:
Done